Dockerfile

```diff
@@ -55,14 +55,20 @@ RUN ["dash", "-c", "\
      bash-completion \
      ca-certificates \
      curl \
+     g++ \
+     gdb \
+     gzip \
+     lcov \
      python-is-python3 \
      python3 \
      python3-pip \
      ssh \
      tree \
+     valgrind \
  && pip install \
      adafruit-nrfutil \
      pyserial \
+    --break-system-packages \
```

Comment:
You get an error about preferring a virtual environment for installing Python packages with the new Debian release unless you let it know you're ok with installing the packages system-wide with `--break-system-packages`.